docs/tutorials/work_with_lsdb.ipynb

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
<p>Were you planning on adding a bit of analysis to this? Maybe just a query and a function run would be helpful</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/nested-dask/pull/18/discussion/#comment-1613809678'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>